pkg/proxy/proxy.go

```diff
@@ -161,12 +161,45 @@ func filterList(rels []*release.Release) []*release.Release {
 	return uniq
 }
 
+func getStatuses(statusQuery string) []release.Status_Code {
```

Comment:
Did we get this function from anywhere?

Reply:
nope, it's quite similar to what the helm CLI does, I can add the reference